backend/games/word_guesser/WordGuesser.py

```diff
@@ -0,0 +1,124 @@
+from enum import IntEnum
+from random import randint
+from typing import Dict, List
+
+MAX_GUESSES = 6
+
+# initialize valid guesses and possible solutions
+SOLUTIONS: str
+VALID_WORDS: str
+
+try:
+    solutions_file = open(
+        "backend/games/word_guesser/WordGuesser_Solutions.txt"
+    )
+    guesses_file = open(
+        "backend/games/word_guesser/WordGuesser_ValidGuesses.txt"
```

Comment:
I had to fix the file paths so that this would work in our production API, make sure to merge the latest code from `main` into your branch.